src/figuresClasses.ts

```diff
@@ -1,19 +1,74 @@
-export interface Figure {
+/*
+  eslint max-len: ["error", { "ignoreStrings": true }]
+*/
 
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle',
+  color: 'red' | 'green' | 'blue',
+  getArea():number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape:Figure['shape'] = 'triangle';
 
-}
+  constructor(
+    public color :Figure['color'],
+    public a :number,
+    public b :number,
+    public c :number,
+  ) {
+    if (a < 0 || b < 0 || c < 0) {
+      throw new Error('set each side to positive number or pick your favorite TS error');
+    }
+
+    if (a >= b + c || b >= a + c || c >= a + b) {
+      throw new Error('this is not a triangle');
```

Comment:
Why not? Let's make this msg more informative. [Checklist](https://github.com/mate-academy/ts_figures_classes/blob/master/checklist.md) the 3rd point